src/Util/Cache/CacheFactory.php

```diff
@@ -89,6 +89,7 @@ public function __construct(
      */
     public function create(?string $cacheKey = null): CacheDriver
     {
+        $cacheKey = (string) $cacheKey;
```

Comment:
The signature is actually already being overwritten by the PHPDoc as only taking strings so maybe this would make more sens.

```suggestion
    public function create(string $cacheKey = ''): CacheDriver
    {
```

I tested it locally and it works fine this way and is a bit less odd.